source/components/passing-properties-to-a-component.md

```diff
@@ -1,97 +1,77 @@
-By default a component does not have access to any properties normally
-available in the template in which it is used (sometimes referred to
-as the template `scope`)
+Components only have access to properties passed to them.
 
-Imagine you have a `blog-post` component that is used to
+For example, imagine you have a `blog-post` component that is used to
 display a blog post:
 
 ```app/templates/components/blog-post.hbs
-<h1>Component: {{title}}</h1>
-<p>Lorem ipsum dolor sit amet.</p>
+<article class="blog-post">
+  <h1>{{title}}</h1>
+  <p>{{body}}</p>
+</article>
 ```
 
-You can see that it has a `{{title}}` Handlebars expression to print the
-value of the `title` property inside the `<h1>`.
-
 Now imagine we have the following template and route:
 
 ```app/routes/index.js
 export default Ember.Route.extend({
   model() {
-    return this.store.findAll('post').get('firstObject');
+    return this.store.findAll('post');
   }
 });
 ```
 
-```app/templates/index.hbs
+If we tried to use the comonent like this:
 
-<h1>Template: {{model.title}}</h1>
-{{blog-post}}
+```app/templates/index.hbs
+{{#each model as |post|}}
+  {{blog-post}}
+{{/each}}
 ```
 
-If you run this code, you will see that the first `<h1>` (in the outer
-template) displays the `title` property, but the second `<h1>` (inside
-the component) is empty.
+The following HTML would be rendered:
 
-We can fix this by making the `title` property available to the
-component:
-
-```handlebars
-{{blog-post title=model.title}}
+```html
+<article class="blog-post">
+  <h1></h1>
+  <p></p>
+</article>
 ```
 
-This will make the `title` property in the outer template scope
-available inside the component's template using the same name, `title`.
-
-If, in the above example, the model's `title` property was instead
-called `name`, we would change the component usage to:
+In order to make a property available to a component, you must pass it
+in like this:
 
-```handlebars
-{{blog-post title=model.name}}
+```app/templates/index.hbs
+{{#each model as |post|}}
+  {{blog-post title=post.title body=post.body}}
+{{/each}}
 ```
 
-In other words, you are binding a named property from the outer scope to
-a named property in the component scope, with the syntax
-`componentProperty=outerProperty`.
-
 It is important to note that these properties stay in sync (technically
 known as being "bound"). That is, if the value of `componentProperty`
 changes in the component, `outerProperty` will be updated to reflect that
 change. The reverse is true as well.
 
-You can also bind properties from inside an `{{#each}}` loop. This will
-create a component for each item and bind it to each model in the loop.
+## Positional Params
+
+Instead of passing parameters in by name, you can pass them in by position.
```

Comment:
you can also use both of them at the same time, this sentence kinda makes it feel XOR